torch/ao/quantization/pt2e/representation/_ops.py

```diff
@@ -0,0 +1,119 @@
+from typing import Tuple
+
+import torch
+from torch.ao.quantization.utils import determine_qparams, validate_qmin_qmax
+from torch.library import impl, Library
+
+quant_lib = Library("quant", "DEF")
```

Comment:
why are we not adding this to quantized_decomposed lib? why a separte lib?

Reply:
that namespace is a bit too long, we may want to deprecate that

Reply:
Ok, so lets pick one. For the comment, ok with the change for now, but lets converge on one name.